helm/security-bundle/values.yaml

```diff
@@ -135,6 +156,17 @@ apps:
     # repo: giantswarm/kyverno-policies
     version: 0.20.2
 
+  reportsServer:
+    appName: reports-server
+    chartName: reports-server
+    catalog: giantswarm-operations
+    dependsOn: cloudnative-pg
```

Comment:
Doesn't this one also need Kyverno?

Reply:
For what?

Reply:
Actually, it doesn't require Kyverno CRDs in the chart, and maybe not even in code, because it's not an operator :thinking: 
We would need to do something similar to the [service mesh bundle ](https://github.com/giantswarm/service-mesh-bundle/blob/f2dc2025e49a94bce6a8129ae526154a4de32412/helm/service-mesh-bundle/templates/apps.yaml#L5)in order to support multiple `dependsOn` in this chart and I'd rather do change separately if needed. WDYT?

Reply:
Yeah, I thought you might need it to patch the api-service. If it's not the case then I would go ahead

Reply:
We'll find out shortly if I forgot something :sweat_smile: Will merge after testing if I get a :+1: